tests/e2e/linalg/BUILD

```diff
@@ -36,7 +36,7 @@ iree_check_single_backend_test_suite(
     name = "check_winograd_llvm-cpu_local-task",
     srcs = LLVM_SRCS,
     compiler_flags = [
-        "--iree-flow-enable-conv-winograd-transform",
+        "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-linalg-ext-convert-conv2d-to-winograd))",
```

Comment:
Downside to this is that these allow for unstable names, don't know if we want to warn about this (not really something easy to do generally, and if things break integrate side it can be changed/no long term stability may be required, the previous approach did allow guarding against that, but fine as known risk).

Reply:
To start with it is just a pass pipeline... How much this is "supported" path depends on how load bearing use of preprocessing becomes (I really really hope that it does not become load bearing, I see that as a failure mode). If we need to support something more robust, I think we can have a more stable approach within the pre-processing.